tests/src/test/scala/cats/tests/SyntaxSuite.scala

```diff
@@ -184,7 +190,6 @@ object SyntaxSuite {
     val mb4: M[B] = ma.parProductR(mb)
     val mab2: M[(A, B)] = ma.parProduct(mb)
     val mb5: M[B] = mab.parAp(ma)
```

Comment:
Would you also add a test for `parProductL` here please? Looks like it didn't exist before, but since you got your hands dirty with this piece of code, it would be nice to improve it too )

Reply:
its already there L189